paper/main.tex

```diff
@@ -706,369 +712,343 @@ \section{Evaluation of performance}\label{section:evaluation_of_performance}
+\(C_{\text{max}}\). These add more emphasis on how important it is for a  a
+strategy to adapt to its environment. Finally, the features number of turns,
+repetitions and the probabilities of noise and the game ending had no
+significant effects based on the multivariate regression models.
 
 \begin{figure}[!htbp]
     \centering
-    \includegraphics[width=.5\textwidth]{../images/compared_to_mean_median_overall.pdf}
-    \caption{Distributions of \(C_r / C_{\text{median}}\)
-    and \(C_r / C_{\text{median}}\) for winners of over all the tournaments.}\label{fig:mean_median_overall}
+        \centering
+        \includegraphics[width=\textwidth]{../images/entire_data_discussion.pdf}
+        \caption{Distributions of \(C_r / C_{\text{mean}}\), SSE and \(CD\) to \(C\) ratio
+        for the winners over the tournaments of the entire data set.}
+        \label{fig:discussion_entire_data}
 \end{figure}
 
-In this section the effect of several features, regarding a strategy's behaviour
-and the tournament in which it participated on its performance were presented.
-This was done using two approaches. Correlation coefficients and a random forest
-analysis. The results of these are summarised in the following section.
-
-\section{Conclusion}\label{section:conclusion}
-
-This manuscript has explored the performance of \numberofstrategies strategies of the Iterated
-Prisoner's Dilemma in a large number of computer tournaments. The results of
-the analysis demonstrated that, although for specific tournament types such as
-standard and probabilistic ending tournaments, dominant strategies exist
-there is not a single dominant type of strategies if the environments
-vary. Moreover, a strategy with a theory of mind should aim to adapt its behaviour
-based on the mean and median cooperators and should in general not be too
-cooperative.
-
-The \numberofstrategies strategies used in this manuscript have been mainly for
-the literature, and they have been accessible due to an open source software
-called the Axelrod-Python library. The software was used to generate a total of
-\numberofalltournaments computer tournaments results with different number of
-strategies and different participants each time. The data collection was
-described in Section~\ref{section:data_collection}. In
-Section~\ref{section:top_performances}, the tournaments results were used to
+A third method that evaluates the importance of the features in
+Table~\ref{table:manual_features} using clustering and random forests can be found in
+the Appendix~\ref{app:clustering}. The results uphold the outcomes of the
+correlation and multivariate regression methods. It also evaluates the effects
+of the classifiers stochastic, make use of game, and make use of length which
+have not been evaluated by the methods above because there are binary variables.
+The results imply that they have no significant effect on a strategy's
+performance.
```

Comment:
```suggestion
of the whether or not a strategy is stochastic, makes use of the knowledge of the utility values, or makes use of match length. These were not evaluated by the methods above because there are binary variables.
The results showed that they have no significant effect on a strategy's
performance.
```